doc/script_commands.txt

```diff
@@ -6119,6 +6119,41 @@ the attached NPC will display the message.
 
 ---------------------------------------
 
+*chatmes "<message>"{,"<NPC name>"};
+
+This command will display a message in the waitingroom (chat) of the npc.
+If the <NPC name> option is given, then that NPC will display the message, else
+the attached NPC will display the message.
+If the NPC is not in a waitingroom, nothing happens.
+
+    // Everyone in the waitingroom will see this message:
+    chatmes "Waiting 5 minutes till the next match will start";
```

Comment:
till -> until or 'til. Till is actually a word of it's own which has no relation to until. Welcome to complications of English!